src/NuGet.Config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="local package" value="Caliburn.Micro.Autofac.Packaging\" />
```

Comment:
We can use pre-release versions from appveyor: https://ci.appveyor.com/nuget/caliburn-micro-6slj1l48j82d